OpenOversight/app/models/database.py

```diff
@@ -39,22 +39,26 @@
 class Department(BaseModel):
     __tablename__ = "departments"
     id = db.Column(db.Integer, primary_key=True)
-    name = db.Column(db.String(255), index=True, unique=True, nullable=False)
+    name = db.Column(db.String(255), index=False, unique=False, nullable=False)
     short_name = db.Column(db.String(100), unique=False, nullable=False)
+    state = db.Column(db.String(2), server_default="", nullable=False)
```

Comment:
It's been set to `db.Column(db.String(2), server_default="", nullable=False)`, @abandoned-prototype.